README.md

```diff
@@ -1,2 +1,23 @@
-# sample-julia-genie
-Julia with Genie Devcontainer Sample
+# Sample Julia App with Genie.jl
+
+This is a sample Julia application using the Genie.jl framework, designed to run in a dev container.
+
+## Setup Instructions
+
+1. Install Docker and VS Code.
+2. Open this folder in VS Code.
+3. Install the "Remote - Containers" extension.
+4. Reopen the project in the dev container.
+5. Install the required dependencies `julia deps.jl`.
+6. Run the app using `julia src/app.jl`.
+
+The application will start at `http://localhost:8000` by default.
+
+## Open using daytona
```

Comment:
`Open using Daytona`

Also, move this section above `Setup instructions` which you can rename to `Setup with VS Code`

Also, since the Daytona section will be first, move step 5. and 6. into the Daytona section.